lib/tokenizers.js

```diff
@@ -185,34 +184,39 @@ async function _readCurrentTokenizerRecord() {
 async function _tokenizerFromRecord({record}) {
   const {tokenizer} = record;
 
-  // 1. Generate capability agent from handle and secret.
+  // 1. Generate capability agent from handle and application identity.
   const handle = 'primary';
-  const secret = base64url.decode(tokenizer.secret);
-  const capabilityAgent = await CapabilityAgent.fromSecret({handle, secret});
+  const {keys: {capabilityInvocationKey}} = getAppIdentity();
+  const keyPair = capabilityInvocationKey;
+  const signer = capabilityInvocationKey.signer();
+  const capabilityAgent = new CapabilityAgent({handle, signer, keyPair});
```

Comment:
Hmm, unless I'm misunderstanding this, this seems like it makes it so that every tokenizer is the same now vs. having N tokenizers per app. We should discuss sometime tomorrow.